### DISCUSSION TITLE
Retrieve all Offers without passing through the policy engine

### DISCUSSION THREAD

**Discussion by galanisd** [eclipse-edc/Connector#4690] [General]
Tested with EDC 0.7.0 and previous releases

When we try to retrieve a dataset or catalog of datasets we use CatalogApiController 
that just requires x-api-key. 

However it seems that the request is internally dispatched
to DspCatalogApiController that a) requires an auth token b) the results are filtered based on what policy engine returns
The auth token is the token of Connector A and contains connectorCountry: Greece

So we have the following scenario, I create an offer X in Connector A with an "access policy" that has a constraint (connectorCountry EQ Germany), because I want only Connectors from Germany to have access to the Offer 

However according to the aforementioned flow [request -> CatalogApiController -> DspCatalogApiController] I can't retrieve my own 
offer X, it is filtered by the policy engine. Only Connectors from Germany are able to get it. So is there a way (endpoint) to retrieve all my offers e.g. without passing through the DspCatalogApiController and the policy engine.

**Comment by ndr-brt** [comment]
EDC 0.7.0 is not supported anymore, please update to the latest version (0.11.x) and verify if the issue is still there
